compute/manifest.yml

```diff
@@ -0,0 +1,27 @@
+pg_settings:
```

Comment:
Let's also add some version-specific and read-only parameters:
1. we set io_combine_limit for PG 17
2. we set recovery_prefetch for replicas/read only > PG 14
3. we set hot_standby for all replicas

Also do not see effective_io_concurrency, protocol versions, etc. from global settings

See `compute->computespec` package in cplane code

I'd prefer to move all static parameters from the Go code and maintain them here

Reply:
We don't need plans in manifest, right. Everything plan-specific should stay in cloud cplane. `neon.max_cluster_size` shouldn't be here

I also forgot to ask you to copy all comments from the Go code around these values. IIRC, there were some valuable clarifications about why we set certain GUC (not for all, ofc